app/cli/src/main/scala/org/bitcoins/cli/ConsoleCli.scala

```diff
@@ -89,6 +93,130 @@ object ConsoleCli {
                 case other => other
               }))
         ),
+      cmd("createdlcoffer")
+        .hidden()
+        .action(
+          (_, conf) =>
+            conf.copy(
+              command = CreateDLCOffer(0.bitcoins,
+                                       null,
```

Comment:
No I don't think it can work here. The `conf` parameter needs to take in an instantiated `CliCommand`, if we use a `X.curried` we are putting in a function, that builds a `CliCommand`. I think this just might be a limitation of `OParser`